purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -796,7 +805,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      */
     @JvmOverloads
     @JvmSynthetic
```

Comment:
This annotation and the one in line 764 should be removed, otherwise it would still be hidden in Java. We add it because internal functions are public in Java, since internal is a Kotlin keyword.